TideApp/TideApp/Assets.xcassets/Colors/backgroundColor.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xF0",
-          "green" : "0xF7",
-          "red" : "0xF9"
+          "blue" : "0.941",
+          "green" : "0.969",
```

Comment:
Not sure why these values changed from  hex to dec??

Reply:
xcode mysteries, but its fine tho